### PR TITLE
Feature/3 create UI designs

### PR DESCRIPTION
**Pull request by YammyToast**

### CODE REVIEW

File: source/OTRServer/public/display_render.js [hunk 1]
```diff
@@ -0,0 +1,275 @@
+// ====================================================================================
+// INIT
+// ====================================================================================
+// Canvas Initialization
+let cnv = document.getElementById("canvas-parent");
+// This is old style initialization apparently.
```
Comment (Mystic0802) [inline source/OTRServer/public/display_render.js:6]: So we don't need this?

File: source/OTRServer/public/display_render.js [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+        }
+        console.log("Adding: ", element[0]);
+        this.elements.set(element[0], element[1]);
+      } catch (e) {
+        console.error(e);
+      }
+    });
+  }
+
+  clear_elements(__element) {
```
Comment (Mystic0802) [inline source/OTRServer/public/display_render.js]: Does this need the parameter?

File: source/OTRServer/public/display_socket.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import display_render from "./display_render.js";
+let socket_path = "/game";
+
+// ====================================================================================
+// State Control
+// ====================================================================================
+
+// State logic enum equivalent.
+const State = Object.freeze({
```
Comment (Mystic0802) [inline source/OTRServer/public/display_socket.js:9]: Can we encapsulate this somewhere and import state in? Reduce the duplicate code etc

**Review by Mystic0802 — COMMENTED** [review]
See other comments

File: source/OTRServer/public/display_render.js (hunk 1 above)
Comment (YammyToast) [reply source/OTRServer/public/display_render.js:6]: Yeah we keep this. 
The documentation for the library is all wack and the latest distribution is running based on what the old documentation is saying, not the new documentation.
It is really awful and it keeps popping up in other areas.

File: source/OTRServer/public/display_render.js (hunk 2 above)
Comment (YammyToast) [reply source/OTRServer/public/display_render.js]: No I will fix this.

File: source/OTRServer/public/display_socket.js (hunk 3 above)
Comment (YammyToast) [reply source/OTRServer/public/display_socket.js:9]: Yeah I can try that.

Given that the display and user screens also need the same state reference i'll try to move it into the public source.
Will be a bit stinky but should work.

**Comment by YammyToast** [comment]
State hasn't been changed in this branch, however it has been fully done in the working #11 one.

**Pull request merged**